hello.php

```diff
@@ -0,0 +1,100 @@
+<?php
```

Comment:
this file doesn't need to be here

Reply:
I hate that Wordpress install adds this plugin. Thought I removed it. I'll remove.